From add1250bcb47566b15daa0303c8e0ec2bf1689c0 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Mon, 15 Jun 2015 10:59:04 +0800
Subject: [PATCH] MLK-11109-2 mxc IPUv3: PRE reg: Correct prefetch engine
 status block Y mask

This patch corrects the macro BM_PRE_PREFETCH_ENGINE_STATUS_PREFETCH_BLOCK_Y
from 0xFFFF0000 to 0x3FFF0000 according to the i.MX6QP reference manual.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 drivers/mxc/ipu3/pre-regs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/ipu3/pre-regs.h b/drivers/mxc/ipu3/pre-regs.h
index 5b2cf93a..6a00f61 100644
--- a/drivers/mxc/ipu3/pre-regs.h
+++ b/drivers/mxc/ipu3/pre-regs.h
@@ -292,7 +292,7 @@
 #define HW_PRE_PREFETCH_ENGINE_STATUS	(0x00000090)
 
 #define BP_PRE_PREFETCH_ENGINE_STATUS_PREFETCH_BLOCK_Y      16
-#define BM_PRE_PREFETCH_ENGINE_STATUS_PREFETCH_BLOCK_Y 0xFFFF0000
+#define BM_PRE_PREFETCH_ENGINE_STATUS_PREFETCH_BLOCK_Y 0x3FFF0000
 #define BF_PRE_PREFETCH_ENGINE_STATUS_PREFETCH_BLOCK_Y(v) \
 	(((v) << 16) & BM_PRE_PREFETCH_ENGINE_STATUS_PREFETCH_BLOCK_Y)
 #define BP_PRE_PREFETCH_ENGINE_STATUS_PREFETCH_BLOCK_X      0
-- 
1.8.0

